docs/samples/browser-plugin-meetings/app.js

```diff
@@ -949,6 +982,31 @@ function getMediaDevices() {
   }
 }
 
+function publishTracks(meeting) {
+  const [localStream] = currentMediaStreams;
+
+  // todo: screen share once it's supported
+
+  if (localStream) {
+    const localAudioTracks = localStream.getAudioTracks();
+
+    if (localAudioTracks?.length > 0) {
+      console.log('MeetingStreams#publishTracks() :: publishing local audio');
+      meeting.media.publishTrack(localAudioTracks[0]);
+    }
+
+    const localVideoTracks = localStream.getVideoTracks();
+
+    if (localVideoTracks?.length > 0) {
+      console.log('MeetingStreams#publishTracks() :: publishing local video');
+      meeting.media.publishTrack(localVideoTracks[0]);
```

Comment:
any thoughts about publishTrack to publishAudioTrack , publishVideoTrack and shareTrack in the next iteration

Reply:
or its easy to publish array of tracks
 ?

Reply:
The details of API for outbound media depend on how the LocalTrack API looks like, so let's keep that discussion out of this PR, because it's not directly related and let's have a separate discussion/meeting/wiki page etc for https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-365637